src/consensus/pbft/libbyz/Replica.cpp

```diff
@@ -832,19 +832,20 @@ void Replica::send_commit(Seqno s)
   }
 
   Commit* c = new Commit(view(), s);
-  send(c, All_replicas);
+  int send_node_id = (send_only_to_self ? node_id : All_replicas);
```

Comment:
Since this code is duplicated (line 806), would it be better to add this logic to `send()` instead?

Reply:
This is specialized to 2 places and only needed when f == 0, this is not something i want to make the general case for all messages